src/deno_json/mod.rs

```diff
@@ -581,115 +612,69 @@ fn decorate_tasks_json(
 }
 
 #[derive(Clone, Debug, Default)]
-pub struct ParseOptions {
+pub struct ConfigParseOptions {
   pub include_task_comments: bool,
 }
 
+#[allow(clippy::disallowed_types)]
```

Comment:
Why is it disallowed type if you're using `MaybeArc` here? Does it switch impl based on a feature flag to use `std::sync::Arc` under the hood?

Reply:
Since it's a type instead of a struct clippy inlines MaybeArc to Arc